recipes/tl-expected/all/test_package/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(PackageTest CXX)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
I'd rather not have this here, but the gcc 5 CI configuration doesn't build the test package as C++11 without it.  

My understanding of `conan_basic_setup` is that it's supposed to set the C++ standard in use.  Am I missing something in the test package?

Reply:
No, conan_basic_setup won't set it. It configures the most artifact paths, libraries and definitions. https://docs.conan.io/en/latest/reference/generators/cmake.html#conan-basic-setup

Reply:
Thanks for the clarification.  I've switched this to use `conan_set_std` in a15986c313ddc9974eb8acdeef0c9dfb3f8d5ffe.